dev-libs/boost/boost-1.72.0-r1.ebuild

```diff
@@ -39,7 +39,15 @@ RDEPEND="
 	mpi? ( >=virtual/mpi-2.0-r4[${MULTILIB_USEDEP},cxx,threads] )
 	python? (
 		${PYTHON_DEPS}
-		numpy? ( dev-python/numpy[${PYTHON_USEDEP}] )
+		numpy? (
+			|| (
```

Comment:
this is actually the wrong way to use numpy, because boost::python links to numpy and should thus bind to it. Given that only one package in the sci overlay relies on numpy support in boost python, I consider an acceptable violation.

Reply:
Can we make it py3-only?

Reply:
yes, do we know what's stopping us?